pkg/operator/ceph/cluster/controller.go

```diff
@@ -253,6 +253,7 @@ func (r *ReconcileCephCluster) reconcile(request reconcile.Request) (reconcile.R
 		return reconcile.Result{}, errors.Wrap(err, "failed to add finalizer")
 	}
 
+	var doCleanup = true
```

Comment:
Shouldn't you initialize this inside the condition? It's less confusing IMO.

Reply:
yes. done.